RELEASE/scripts/autoscend/auto_acquire.ash

```diff
@@ -478,7 +478,8 @@ boolean auto_hermit(int amt, item it)
 		return hermit(amt, it);
 	}
 	int initial = item_amount(it);
-	catch hermit(amt, it);
+	//store the result to avoid an error being thrown
+	boolean result = hermit(amt, it);
```

Comment:
from my tests using `boolean result` has the same effect as `catch`

Reply:
In that it doesn't suppress the error? Unable to check myself, but I think I was under the impression that boolean will suppress any error instead.

Reply:
> In that it doesn't suppress the error? Unable to check myself, but I think I was under the impression that boolean will suppress any error instead.

you can test it by running mafia and using
`ash boolean result = hermit(1, $item[11-leaf clover]); print("did not abort");`
`ash catch hermit(1, $item[11-leaf clover]); print("did not abort");`

in both cases it flickers red as mafia sets error state during hermit command, but then continues on to perform the next operation and prints "did not abort" and turn green.
if you had neither
`ash hermit(1, $item[11-leaf clover]); print("did not abort");`
will result in NOT printing "did not abort" as it aborted when hermit fails.

Reply:
```suggestion
	catch hermit(amt, it);
```